tests/PKSim.Matlab.Tests/PKSim.Matlab.Tests.csproj

```diff
@@ -74,5 +71,16 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-  
+  <PropertyGroup>
+    <ContentSQLiteInteropFiles>true</ContentSQLiteInteropFiles>
+    <CopySQLiteInteropFiles>false</CopySQLiteInteropFiles>
+    <CleanSQLiteInteropFiles>false</CleanSQLiteInteropFiles>
+    <CollectSQLiteInteropFiles>false</CollectSQLiteInteropFiles>
+  </PropertyGroup>
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <ItemGroup>
+      <WindowsFiles Include="$(TargetDir)x64/SQLite.Interop.dll" />
+    </ItemGroup>
+    <Copy SourceFiles="@(WindowsFiles);" DestinationFolder="$(TargetFolder)" DestinationFiles="@(WindowsFiles->Replace('x64/SQLite.Interop.dll', 'SQLite.Interop.dll'))" />
+  </Target>
```

Comment:
For tests, we need to get this SQLite.Interop.dll into the root. This just copies the x64 version up one level to the output folder

Reply:
For now, no test projects are running on Linux, and other than debugging, we don't really need to build on Linux anyway.